.gitignore

```diff
@@ -5,4 +5,7 @@
 **/.idea/**
 
 # transpiled .js files
-packages/**/dist
\ No newline at end of file
+packages/**/dist
+
+# coverage reports
+**/coverage/**
```

Comment:
Is this one intentional?

Reply:
Yes, because I make a npx jest --coverage that created a lot of documents.
I do this to look at which parts of the code are not covered by unit tests

Reply:
I'm good with it, but it will be good to comment it to the rest of the team.